src/App.fs

```diff
@@ -27,10 +27,18 @@ type Msg =
     | NewTodoChanged of string
     | AddNewTodo
     | DeleteTodo of TodoId
+    | CompleteTodo of TodoId * bool
```

Comment:
Personally, I am not a big fan of this boolean flag. I would have created two different messages: `CompleteTodo` and `UncompleteTodo`. I feel there is value in knowing what exactly happened and not having to juggle booleans